src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const getObjectCopy = { ...state };
```

Comment:
getObjectCopy is more suitable as function name.
```suggestion
  const stateCopy = { ...state };
```